### ISSUE TITLE
Server log spam due to null id, related to timer

### ISSUE THREAD

**Issue by xenohedron** [magefree/mage#11876]
No apparent impact to any match, but every match is logging this at what seems to be every priority pass, on both PAUSE_TIMER and RESUME_TIMER

This inflated the server logs to 1.5MB over the course of one draft tournament.

```
FATAL 2024-02-29 02:48:22,180 Table event listener error                                                                 =>[GAME 7375ed49-bf5c-409b-803e-dd51280fc059] GameController.lambda$init$3aa0d15c$1 
mage.MageException: RESUME_TIMER: playerId can't be null
        at mage.server.game.GameController.lambda$init$3aa0d15c$1(GameController.java:149)
        at mage.game.events.EventDispatcher.fireEvent(EventDispatcher.java:28)
        at mage.game.events.TableEventSource.fireTableEvent(TableEventSource.java:52)
        at mage.game.GameImpl.resumeTimer(GameImpl.java:3805)
        at mage.player.human.HumanPlayer.waitForResponse(HumanPlayer.java:316)
        at mage.player.human.HumanPlayer.choose(HumanPlayer.java:673)
        at mage.player.human.HumanPlayer.choose(HumanPlayer.java:632)
        at mage.game.GameImpl.init(GameImpl.java:1258)
        at mage.game.TwoPlayerDuel.init(TwoPlayerDuel.java:35)
        at mage.game.GameImpl.start(GameImpl.java:999)
        at mage.server.game.GameWorker.call(GameWorker.java:34)
        at mage.server.game.GameWorker.call(GameWorker.java:15)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:750)

```

**Comment by JayDi85** [comment]
Fixed in #11873

**Comment by JayDi85** [comment]
Correct commit with timers logs fix: 229e8d307554be9e9aeb9ae9fa345f511547a802

**Comment by JayDi85** [comment]
It's not affect a real games (only AI related), so you can temporary comment that logs.
